DQM/TrackingMonitor/src/TrackSplittingMonitor.cc

```diff
@@ -148,15 +156,6 @@ void TrackSplittingMonitor::bookHistograms(DQMStore::IBooker & ibooker,
 void TrackSplittingMonitor::beginJob(void) 
```

Comment:
yes
... if you add it to beginJob, the compiler will tell you that it's not a method inherited from the base class, which is a pretty good indicator that the method is not right (and in this case should be removed)